<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OISettingsDialog</class>
 <widget class="QWidget" name="OISettingsDialog">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>OpenStreetMap Importer Settings</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <widget class="QFrame" name="frame">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>The OpenStreetMap Importer reads an OSM file. Reading from an OSM XML file can take quite some time due to slow XML reading. Intermediate data is cached in the output directory in a compressed file format.</string>
        </property>
        <property name="scaledContents">
         <bool>false</bool>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QToolBox" name="toolBox">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="page_4">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>303</width>
        <height>152</height>
       </rect>
      </property>
      <attribute name="label">
       <string>Options</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Input File</string>
           </property>
           <property name="buddy">
            <cstring>inputEdit</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="inputEdit"/>
         </item>
         <item>
          <widget class="QPushButton" name="browseButton">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Browse...</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Speed Profile</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="speedProfileChooser"/>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QCheckBox" name="customProfile">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Custom Speed Profile</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_5">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>100</width>
        <height>131</height>
       </rect>
      </property>
      <attribute name="label">
       <string>Name Language Settings</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_8">
         <item>
          <widget class="QToolButton" name="addLanguage">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>+</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="deleteLanguage">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string> - </string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_6">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QListWidget" name="languagePriorities"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>437</width>
        <height>204</height>
       </rect>
      </property>
      <attribute name="label">
       <string>Speed Profile: Settings</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <item>
          <widget class="QPushButton" name="saveButton">
           <property name="text">
            <string>Save</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="loadButton">
           <property name="text">
            <string>Load</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_5">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Traffic Light Penalty</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="trafficLightPenalty">
           <property name="frame">
            <bool>true</bool>
           </property>
           <property name="suffix">
            <string> s</string>
           </property>
           <property name="maximum">
            <number>999</number>
           </property>
           <property name="value">
            <number>5</number>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QCheckBox" name="setDefaultCitySpeed">
         <property name="text">
          <string>Determine Speed within Cities with approximate City Boundaries</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="ignoreOneway">
         <property name="text">
          <string>Ignore Oneway Tags</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="ignoreMaxspeed">
         <property name="text">
          <string>Ignore Maxspeed Tag</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>100</width>
        <height>131</height>
       </rect>
      </property>
      <attribute name="label">
       <string>Speed Profile: Way Types</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QToolButton" name="addWayType">
           <property name="text">
            <string>+</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="deleteWayType">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string> - </string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTableWidget" name="speedTable">
         <property name="columnCount">
          <number>4</number>
         </property>
         <attribute name="horizontalHeaderDefaultSectionSize">
          <number>150</number>
         </attribute>
         <column>
          <property name="text">
           <string>Highway Type</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Default Speed</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Default City Speed</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Average Speed</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_3">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>100</width>
        <height>100</height>
       </rect>
      </property>
      <attribute name="label">
       <string>Speed Profile: Access</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QTreeWidget" name="accessTree">
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <attribute name="headerVisible">
          <bool>false</bool>
         </attribute>
         <column>
          <property name="text">
           <string>Transportation Type</string>
          </property>
         </column>
         <item>
          <property name="text">
           <string>access</string>
          </property>
          <item>
           <property name="text">
            <string>foot</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>ski</string>
           </property>
           <item>
            <property name="text">
             <string>ski:nordic</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>ski:alpine</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>ski:telemark</string>
            </property>
           </item>
          </item>
          <item>
           <property name="text">
            <string>inline_skates</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>ice_skates</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>climbing</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>hiking</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>horse</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>vehicle</string>
           </property>
           <item>
            <property name="text">
             <string>bicycle</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>carriage</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>wheelchair</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>motor_vehicle</string>
            </property>
            <item>
             <property name="text">
              <string>motorcylce</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>moped</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>mofa</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>motorcar</string>
             </property>
             <item>
              <property name="text">
               <string>hov</string>
              </property>
             </item>
            </item>
            <item>
             <property name="text">
              <string>caravan</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>goods</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>psv</string>
             </property>
             <item>
              <property name="text">
               <string>bus</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>taxi</string>
              </property>
             </item>
            </item>
            <item>
             <property name="text">
              <string>snowmobile</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>agricultural</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>forestry</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>emergency</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>hazmat</string>
             </property>
            </item>
           </item>
          </item>
         </item>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
